### ISSUE TITLE
[BUG] Drag&Drop import resources does't works on selected resources 

### ISSUE THREAD

**Issue by edesdan** [UnderwaterApps/overlap2d#253]
Let's say we have a folder that contains these files:
rabbitGreen1
rabbitGreen2
rabbitGreen3
rabbitWhite1
rabbitWhite2
rabbitWhite3

and I want to import only rabbitGreen (selecting just rabbitGreen1-rabbitGreen2-rabbitGreen3). When I drop the 3 files into the DROP resources window the whole directory gets imported, resulting in a single animation of a rabbit that change color from green to white... 

[version]: 0.1.1
[os]: windows 7

**Comment by azakhary** [comment]
Yeah I know, encountered that one. and not sure how to fix. I kinda do wanted import dialog to be "smart" and know what is it that you import. now to solve this scenario, when you import animation, it has to always ask "are you sure this is animation?" which sucks... so I am not sure what is the right thing to do here. If there is an awesome creative solution someone has about how to make it work but still keep super user friendly let me know. if now. I will have to put up the are you sure dialog.

**Comment by CoderJoe1999** [comment]
Maybe the dialog could list the images about to be imported.
